examples/upgrade/blue-green-route53/core-infra/locals.tf

```diff
@@ -0,0 +1,15 @@
+locals {
```

Comment:
We are moving away from a separate`locals.tf`. Can you please combine this with `main.tf`?

Reply:
ok